piet-wgsl/examples/winit/src/main.rs

```diff
@@ -19,25 +19,26 @@ mod simple_text;
 mod test_scene;
 
 use piet_scene::{Scene, SceneBuilder};
-use piet_wgsl::{util::RenderContext, Renderer, Result};
+use piet_wgsl::{util::RenderContext, Renderer};
+use winit::{event_loop::EventLoop, window::Window};
 
-async fn run() -> Result<()> {
-    use winit::{
-        dpi::LogicalSize,
-        event::*,
-        event_loop::{ControlFlow, EventLoop},
-        window::WindowBuilder,
-    };
+async fn run() {
+    use winit::{dpi::LogicalSize, window::WindowBuilder};
     let event_loop = EventLoop::new();
     let window = WindowBuilder::new()
         .with_inner_size(LogicalSize::new(1044, 800))
         .with_resizable(true)
         .build(&event_loop)
         .unwrap();
-    let render_cx = RenderContext::new().await?;
+    run_wasm(event_loop, window).await
+}
+
+async fn run_wasm(event_loop: EventLoop<()>, window: Window) {
```

Comment:
This function name isn't quite right. I'd be tempted here to factor out the wasm specific code from `main` into the equivalent place in `run`, and then not create the second function. 

AFAIK, `resizable` just does nothing on web, rather than causing issues.

Reply:
Refactored this a bit: `run_wasm` -> `run` and the native code is simply pulled into `main`